cores/esp8266/core_esp8266_waveform_phase.cpp

```diff
@@ -267,7 +264,7 @@ static IRAM_ATTR void timer1Interrupt() {
   const bool isCPU2X = CPU2X & 1;
   if ((waveform.toSetBits && !(waveform.enabled & waveform.toSetBits)) || waveform.toDisableBits) {
     // Handle enable/disable requests from main app.
-    waveform.enabled = (waveform.enabled & ~waveform.toDisableBits) | waveform.toSetBits; // Set the requested waveforms on/off
+    waveform.enabled = (waveform.enabled | waveform.toSetBits) & ~waveform.toDisableBits; // Set the requested waveforms on/off
```

Comment:
I suppose there is a slight logic change here if the "toset" and "todisable" bitmasks overlap (which seems like a programming error somewhere higher in the stack).  Intended?  Fixing something you found?

Reply:
It's been a while now, but I still remember actually doing this change to fix a bug.